frontend/src/app/account-activation/account-activation.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AccountActivationRequestModel } from 'app/auth.models';
+import { Observable } from 'rxjs/Observable';
+import { AuthService } from '../auth.service';
+import { Router, ActivatedRoute, Params } from '@angular/router';
+
+@Component({
+  selector: 'volontulo-account-activation',
+  templateUrl: './account-activation.component.html',
+  styleUrls: ['./account-activation.component.css']
+})
+export class AccountActivationComponent implements OnInit {
+  accountActivationRequestModel: AccountActivationRequestModel = {
```

Comment:
You can use a lot shorter names here - it will be recognized also using component, that store it